tests/pauli/test_pauli_arithmetic.py

```diff
@@ -393,7 +393,10 @@ def test_to_mat_format(self, pw, wire_order, true_matrix):
     @pytest.mark.parametrize("pw, op", tup_pw_operation)
     def test_operation(self, pw, op):
         """Test that a PauliWord can be cast to a PL operation."""
-        pw_op = pw.operation()
```

Comment:
Just added some queuing checks across a number of tests.
Let me know if it should get its own test.